Assets/Scripts/Models/Furniture.cs

```diff
@@ -337,6 +415,10 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
         return obj;
     }
 
+    /// <summary>
+    /// The update function used by unity. Updates every frame
+    /// </summary>
+    /// <param name="deltaTime">The time since the last update was called.</param>
     public void Update(float deltaTime)
```

Comment:
Aaaarg fudge. I relized that, i was just busy with multiple stuff. Feel
free to edit it. I am in bed now, if not fixed, i will fix it tomorrow
morning.

On Sun, 04 Sep 2016, 22:44 Gleb <notifications@github.com> wrote:

> In Assets/Scripts/Models/Furniture.cs
> <https://github.com/TeamPorcupine/ProjectPorcupine/pull/1136#discussion_r77458313>
> :
>
> > @@ -337,6 +415,10 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
> >          return obj;
> >      }
> >
> > +    /// <summary>
> > +    /// The update function used by unity. Updates every frame
> > +    /// </summary>
> > +    /// <param name="deltaTime">The time since the last update was called.</param>
> >      public void Update(float deltaTime)
>
> This Update method is not used by Unity.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TeamPorcupine/ProjectPorcupine/pull/1136/files/f7cefa83e33f4c95cc7a4b0b362d4e30fd2e4538#r77458313>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADcrFOos2Sd9YRJ0aWcDpQuIrpsEjQSTks5qmy2agaJpZM4J0nYH>
> .
>